tutorials/interactive/index.md

```diff
@@ -0,0 +1,781 @@
+---
+layout: tutorial
+title: "Adding Interactivity to your Flutter App"
+sidebar: home_sidebar
+permalink: /tutorials/interactive/
+---
+
+{% comment %}
+TODO:
+
+* Restore _private variables in tapbox examples.
+* Add the following examples to the GitHub repo, and update all links
+  * lakes-interactive
+  * tapbox-a
+  * tapbox-b
+  * tapbox-c
+{% endcomment %}
+
+## Introduction
+
+How do you modify your app to make it react to user input? In
+[Building Layouts for Flutter](https://flutter.io/tutorials/layout/),
+we showed how to create the layout for the Lakes app:
+
+<center><img src="images/lakes.png" style="border:1px solid black" alt="The starting Lakes app that we will modify"></center><br>
```

Comment:
Let's use CSS to center and create space after the image. I think bootstrap even has a class to center something.